spec/features/managing_user_memberships_spec.rb

```diff
@@ -1,26 +1,42 @@
 require 'rails_helper'
 RSpec.feature 'Managing user memberships' do
+  let!(:user) { FactoryBot.create(:user, suppliers: [supplier_1]) }
```

Comment:
I've never really liked using `let`, but we seem to be using this now, so :+1:

Reply:
let's (😁) have a chat about this some time. if there's a clearer/better/preferred way to define memoized variables in the specs I'm all for exploring it..